### PR TITLE
Add API docs for State Version Outputs

### PR DESCRIPTION
**Pull request by justincampbell**

### CODE REVIEW

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 1]
```diff
@@ -290,10 +306,90 @@ curl \
                     "related": "/api/v2/runs/sv-SDboVZC8TCxXEneJ/created-by"
                 }
             }
```
Comment (findkim) [inline content/source/docs/enterprise/api/state-versions.html.md]: Missing comma!

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 2]
```diff
@@ -290,10 +306,90 @@ curl \
...
+            "created-by": {
+                "data": {
+                    "id": "api-org-hashicorp",
+                    "type": "users"
+                },
+                "links": {
+                    "related": "/api/v2/runs/sv-SDboVZC8TCxXEneJ/created-by"
+                }
+            }
+            "outputs": {
```
Comment (findkim) [inline content/source/docs/enterprise/api/state-versions.html.md]: missing comma before "outputs"!

File: content/source/docs/enterprise/api/state-version-outputs.html.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+sidebar_current: "docs-enterprise2-api-state-version-outputs"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: /docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# State Version Outputs API
+
```
Comment (findkim) [inline content/source/docs/enterprise/api/state-version-outputs.html.md]: Can you include a brief description as to what a state version output is?

Comment (justincampbell) [reply content/source/docs/enterprise/api/state-version-outputs.html.md]: Added!

File: content/source/docs/enterprise/api/state-version-outputs.html.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+State Version Outputs are the outputs from a Terraform state file. They include
+the name and value of the output, as well as a sensitive boolean if the value
+should be hidden by default in UIs.
+
+## Show a State Version Output
+
+`GET /state-version-outputs/:state_version_output_id`
```
Comment (nfagerlund) [inline content/source/docs/enterprise/api/state-version-outputs.html.md]:
```suggestion
`GET /state-version-outputs/:state_version_output_id`

Parameter | Description
----------|---------
`:state_version_output_id` | The ID of the desired state version output.

State version output IDs must be obtained from a [state version object](./state-versions.html). When requesting a state version, you can optionally add `?include=outputs` to include full details for all of that state version's outputs. 
```

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 5]
```diff
@@ -289,6 +305,80 @@ curl \
...
+        "links": {
+            "self": "/api/v2/state-versions/sv-SDboVZC8TCxXEneJ"
+        }
+    }
+}
+```
+
+## Show a State Version
+
+`GET /state-versions/:state_version_id
```
Comment (nfagerlund) [inline content/source/docs/enterprise/api/state-versions.html.md]:
```suggestion
`GET /state-versions/:state_version_id

Parameter | Description
----------|---------
`:state_version_id` | The ID of the desired state version.
```

File: content/source/docs/enterprise/api/state-version-outputs.html.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+---
+layout: enterprise2
+page_title: "State Version Outputs - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-state-version-outputs"
```
[the remark this answers is not in the document]
Comment (justincampbell) [reply content/source/docs/enterprise/api/state-version-outputs.html.md]: Ok! I copied this from another template, will remove.

**Pull request merged**